05 web-portfolio/index.html

```
@@ -0,0 +1,55 @@
+    <section class="dream-area">
+        <div class="half-width">
+            <img src="images/mahdi1.png" alt="">
+        </div>
+        <div class="half-width">
+            <h2>Dream Big</h2>
+            <h3>Become a web developer</h3>
+            <p>I already learned the basic HTML and CSS. I can build any simple website. I can evev teach my friends how to make simple website. My goal is to build 3 website and learn advanced topics.</p>
+            <a href="https://drive.google.com/drive/" target="_blank" class="link-button">DOWNLOAD RESUME</a>
+        </div>
+    </section>
+    <SECTIon class="experience-area">
```

HTMLHint (1.5.0)

[error] The html element name of [ SECTIon ] must be in lowercase.

(tagname-lowercase)